pkg/shared/ruleset/disak8sstig/rules/242383.go

```diff
@@ -37,19 +37,35 @@ type Options242383 struct {
 var _ option.Option = (*Options242383)(nil)
 
 type AcceptedResources242383 struct {
-	SelectResource
+	ResourceSelector
 	Justification string `json:"justification" yaml:"justification"`
 	Status        string `json:"status" yaml:"status"`
 }
 
-type SelectResource struct {
-	APIVersion     string            `json:"apiVersion" yaml:"apiVersion"`
-	Kind           string            `json:"kind" yaml:"kind"`
-	MatchLabels    map[string]string `json:"matchLabels" yaml:"matchLabels"`
-	NamespaceNames []string          `json:"namespaceNames" yaml:"namespaceNames"`
+func (o Options242383) Validate() field.ErrorList {
+	var (
+		allErrs  field.ErrorList
+		rootPath = field.NewPath("acceptedResources")
+	)
+	for _, p := range o.AcceptedResources {
+		allErrs = append(allErrs, p.Validate()...)
+		if !slices.Contains([]string{"Passed", "Accepted"}, p.Status) && len(p.Status) > 0 {
+			allErrs = append(allErrs, field.Invalid(rootPath.Child("status"), p.Status, "must be one of 'Passed' or 'Accepted'"))
+		}
+	}
+	return allErrs
 }
 
-func (o Options242383) Validate() field.ErrorList {
+type ResourceSelector struct {
```

Comment:
Let's rename this to ObjectSelector but leave the exposed API using the "acceptedResources" naming since we do not want to make another breaking change
```suggestion
type ObjectSelector struct {
```